app/src/main/java/com/x8bit/bitwarden/data/vault/repository/VaultRepository.kt

```diff
@@ -167,6 +167,18 @@ interface VaultRepository : CipherManager, VaultLockManager {
      */
     fun emitTotpCodeResult(totpCodeResult: TotpCodeResult)
 
+    /**
+     * Attempt to unlock the vault using the stored authenticator sync unlock key for the
+     * given user.
+     *
+     * @param userId ID of the user's vault to unlock.
+     * @param decryptedUserKey The authenticator sync unlock key for the user.
```

Comment:
Can we maybe combine the general and specific approaches and do something like

```kotlin
     * @param decryptedUserKey A decrypted user key for the user (ex: their authenticator sync unlock key)
```

Reply:
Yeah, I totally see what you mean here...

What do you think of this:

780faf2d099ef9b9e4e2cd43922691cc77c57fa7

Keeps things generic to match the vault sdk and also to allow for future functionality, but leaves hints about what this is currently used for.

Reply:
Looks good!